forte/base_classes/state_info.cc

```diff
@@ -114,11 +114,12 @@ StateInfo make_state_info_from_psi(std::shared_ptr<ForteOptions> options) {
     //    triplet: multiplicity = 3 -> twice_ms = 0 (ms = 0)
     size_t twice_ms = (multiplicity + 1) % 2;
     if (not options->is_none("MS")) {
-        twice_ms = std::round(2.0 * options->get_double("MS"));
+        twice_ms = std::lround(2.0 * options->get_double("MS"));
```

Comment:
This was causing an error on my computer. Going to `lround` seems the right thing to do anyway.